### DISCUSSION TITLE
Question about VASP calculations

### DISCUSSION THREAD

**Discussion by Claudia-Hello** [CederGroupHub/chgnet#112] [Q&A]
Hi all,
I wonder the whole process of VASP calculation in CHGNet, including the input and the output, since I don't know how to produce the VASP outputs that can be parsed in function "parse_vasp_dir". Thank you so much!

**Comment by bowen-bd** [comment]
VASP is an ab initio calculation software, which contributes training data to CHGNet.
If you want to learn VASP, lease refer to some [VASP tutorials](https://www.youtube.com/watch?v=WiyQUHphYJo)
